### DISCUSSION TITLE
Data storage location?

### DISCUSSION THREAD

**Discussion by semi710** [Akylas/OSS-DocumentScanner#100] [Q&A] [answered]
I am trying to use foldersync to backup my data on clound folder.
but unable to figure out where does the app stores it's data?

**Comment by farfromrefug** [comment]
@niksingh710 if you have an sdcard:
* /storage/sdcard/Android/data/com.akylas.documentscanner/files

else 
* /storage/emulated/0/Android/data/com.akylas.documentscanner/files

**Reply by semi710** [reply]
i am using internal storage.
and i have scanned a 700 kb file then also the files dir is blank.

**Reply by farfromrefug** [reply]
@niksingh710 not sure about your phone setup. but it is the normal. location. in the files folder there should be a `db.sqlite` file.

**Reply by semi710** [reply]
could be a possibility that android is not showing that as accessing that folder requires root.

**Comment by semi710** [comment]
wouldn't it be possible to have an explicit choice of folder to where store the data.
so that the user can keep the data their and then it's up to user to take the dir's backup.

as ig `Android/data/com.xxx.xx/` is not accessible to other apps.

**Comment by farfromrefug** [comment, marked as answer]
@niksingh710 yes that is a real possibility.
if that is the case you cant do it.
I will look at integrating foldersync in the app. but I can't tell you when
you can also look at WebDAV sync which is already included in the app

**Reply by semi710** [reply]
looking forward towards the addition.
i have looked about WebDAV Sync but that requires me owning an server and rn m not having one.

**Reply by Nionor** [reply]
@farfromrefug would it be possible to have a custom location as an option also? 
That way you can choose any kind of external sync/backup service you like.

**Reply by farfromrefug** [reply]
@Nionor it was there at one point but was not working, Will have another go at it. The pain is Android Storage Access Framework in latest versions
can you open a feature request for it?

**Reply by Nionor** [reply]
Sure thing! #184
